index.js

```diff
@@ -103,7 +121,7 @@ WinstonDynamoDB.prototype.submit = function (callback) {
         this.logStreamName() : this.logStreamName;
 
     if (isEmpty(this.logEvents)) {
-        return callback();
```

Comment:
Please revert all of the changes to `callback()` to `callback(null, true)`. I'm not sure what the implications of this, but we seem to only do this when there's an internal winston exception, so we should keep it that way

Reply:
cdca7a6f06b2b3da273410d2acee79eeff30aa99